spec/cc/engine/content_resolver_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+require "rubocop"
+require "cc/engine/content_resolver"
+
+module CC::Engine
+  describe ContentResolver do
+    cops = RuboCop::Cop::Cop.all
+    whitelist = File.read("./spec/support/docs_whitelist.txt").lines.map(&:chomp)
+
+    it "has cops" do
+      expect(cops.count).to eq 310
```

Comment:
That's true. I added this because I wanted to make sure `RuboCop::Cop::Cop.all` didn't return empty array, because if it did than the other tests wouldn't run at all. There's no reason to think that will happen, it was just a sanity check, mostly. WDYT about keeping it, but making it less specific? Happy to just get rid of it as well